src/modules/fundingManager/bondingCurveFundingManager/marketMaker/IVirtualCollateralSupply.sol

```diff
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+
+pragma solidity 0.8.19;
+
+interface IVirtualCollateralSupply {
```

Comment:
I would add the getters here too

Reply:
If you want to Force the usage of modifiers in these functions you can instead create virtual/abstract functions
maybe in that regard you want to use super.setVirtualCollateralSupply()

Reply:
Done